src/pybind/mgr/dashboard/run-frontend-e2e-tests.sh

```diff
@@ -2,29 +2,40 @@
 
 set -e
 
+$CLUSTERS=("c1","c2")
+
 start_ceph() {
     cd $FULL_PATH_BUILD_DIR
 
-    MGR=2 RGW=1 ../src/vstart.sh -n -d
-    sleep 10
+    for cluster in ${CLUSTERS[@]}; do
+        MGR=2 RGW=1 ../src/mstart.sh $cluster -n -d
+        sleep 10
+    done
 
     set -x
+    alias ceph='../src/mrun c1 ceph'
+    alias ceph2='../src/mrun c2 ceph'
 
     # Create an Object Gateway User
-    ./bin/ceph dashboard set-rgw-credentials
+    ceph dashboard set-rgw-credentials
+    ceph2 dashboard set-rgw-credentials
```

Comment:
What about a function called `ceph_all()` that calls the same command in every cluster?

```suggestion
    ceph_all dashboard set-rgw-credentials
```

Reply:
Done, thanks